### PR TITLE
Makefile refactoring

### PR DESCRIPTION
**Pull request by AntoineCezar**

### CODE REVIEW

**Comment by leplatrem** [comment]
Just one remark, not related to your changes : make serve from scratch does not run. make install has to be ran previously

**Comment by AntoineCezar** [comment]
Good remark. It should works now

**Comment by leplatrem** [comment]
Nothing against merging this on my side

File: Makefile [hunk 1]
```diff
@@ -10,25 +12,24 @@ OBJECTS = bin/ lib/ local/ include/ man/ .coverage d2to1-0.2.7-py2.7.egg \
 
 all: install
 install: $(INSTALL_STAMP)
-
-install-dev: $(DEV_STAMP)
-
-$(INSTALL_STAMP): $(VENV_STAMP)
-	bin/python setup.py develop
+$(INSTALL_STAMP): virtualenv
```
Comment (almet) [inline Makefile:15]: shouldn't this be $VIRTUALENV (in case it's over-written)?

**Comment by almet** [comment]
+1 otherwise.

File: Makefile (hunk 1 above)
Comment (AntoineCezar) [inline Makefile:15]: Its not related to $VIRTUALENV. It could have been done this way:

```
$(INSTALL_STAMP): $(PYTHON)
```

**Pull request merged**